node/node.go

```diff
@@ -318,6 +319,51 @@ func (n *Node) GetValueForKey(address string, key string, options api.AccountQue
 	return hex.EncodeToString(valueBytes), blockInfo, nil
 }
 
+// GetGuardianData returns the guardian data for given account
+func (n *Node) GetGuardianData(address string, options api.AccountQueryOptions) (api.GuardianData, api.BlockInfo, error) {
```

Comment:
can we add a unit test for this as well?

Reply:
added